gc/base/GCExtensionsBase.hpp

```diff
@@ -70,6 +70,7 @@ class MM_ReferenceChainWalkerMarkMap;
 class MM_RememberedSetCardBucket;
 #if defined(OMR_GC_STACCATO)
 class MM_RememberedSetWorkPackets;
+class MM_RememberedSetSATB;
```

Comment:
Right, I believe there is no difference between the two. But I think we should remove both and replace with something like OMR_GC_SATB_BARRIER. It would be a separate change.